metpy/calc/tests/test_kinematics.py

```diff
@@ -18,7 +19,7 @@ def test_default_order_warns():
     """Test that using the default array ordering issues a warning."""
     u = np.ones((3, 3)) * units('m/s')
     with pytest.warns(FutureWarning):
-        convergence_vorticity(u, u, 1 * units.meter, 1 * units.meter)
+        c, v = convergence_vorticity(u, u, 1 * units.meter, 1 * units.meter)
```

Comment:
I think this needs to rebase on latest master or something--this line shouldn't be coming back.